src/preamble.ts

```diff
@@ -1,10 +1,12 @@
-export const Preamble = `
+// Placeholders to keep typescript happy
+type $Atomic = ''
+const $InputTypes: Record<string, any> = {}
+
+/** BASE_HERE **/
+
 import { TypedDocumentNode } from '@graphql-typed-document-node/core'
 import gql from 'graphql-tag'
 
-/* tslint:disable */
-/* eslint-disable */
```

Comment:
These we probably want to include/keep somehow as conflicting eslint rules exist and the target user may be using some that don't match ours.

Reply:
Oops yeah I should not have deleted that part.